website/docs/r/wafv2_web_acl.html.markdown

```diff
@@ -465,6 +465,7 @@ This resource supports the following arguments:
 * `description` - (Optional) Friendly description of the WebACL.
 * `name` - (Required, Forces new resource) Friendly name of the WebACL.
 * `rule` - (Optional) Rule blocks used to identify the web requests that you want to `allow`, `block`, or `count`. See [`rule`](#rule-block) below for details.
+* `rule_json` (Optional) Raw JSON block to allow more than three rules. Conflicts with `rule` attribute. **NOTE There is no drift detection at this time. If you use this attribute instead of `rule`, you will be foregoing drift detection.
```

Comment:
Maybe mention that this attribute is for advanced use cases when there are more than 3 levels of nested statements.
Also, can we provide a link to the JSON structure, e.g. https://docs.aws.amazon.com/waf/latest/APIReference/API_CreateWebACL.html?